NavigateU/NavigateU/EditProfileScreen/EditProfileViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  EditProfileViewController.swift
+//  NavigateU
+//
+//  Created by Faki Doosuur Doris on 15.08.2024.
+//
+
+import UIKit
+
+class EditProfileViewController: UIViewController, EditProfileDelegate {
+
+    var editProfileView: EditProfileView?
+    let viewModel: EditProfileViewModel
```

Comment:
please, use protocols for describing properties, it's the one of the most important architecture principle